### DISCUSSION TITLE
Falcon and Kyber OIDs

### DISCUSSION THREAD

**Discussion by likakisp** [IETF-Hackathon/pqc-certificates#47] [General]
Good morning.

Should the Falcon OIDs be updated to the latest OIDs?
1.3.9999.3.1  ->  1.3.9999.3.6
1.3.9999.3.4  ->  1.3.9999.3.9

From this link, it looks like they were updated to .6 and .9 respectively:
https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_1_1-stable/include/openssl/obj_mac.h

#define SN_falcon512            "falcon512"
#define LN_falcon512            "falcon512"
#define NID_falcon512           1247
#define OBJ_falcon512           1L,3L,9999L,3L,6L

#define SN_falcon1024           "falcon1024"
#define LN_falcon1024           "falcon1024"
#define NID_falcon1024          1250
#define OBJ_falcon1024          1L,3L,9999L,3L,9L

**Kyber**:
Are you planning on adding the Kyber OIDs based on this draft?
From: https://datatracker.ietf.org/doc/html/draft-uni-qsckeys-00.html#name-kyber

1.3.6.1.4.1.2.267.8.2.2   (kyber-512-r3)
1.3.6.1.4.1.2.267.8.3.3   (kyber-768-r3)  (noted as '1.3.6.1.4.1.2.267.8.3' on website; **probably a typo**).
1.3.6.1.4.1.2.267.8.4.4   (kyber-1024-r3)
1.3.6.1.4.1.2.267.10.2.2  (kyber-512-90s-r3)
1.3.6.1.4.1.2.267.10.3.3  (kyber-768-90s-r3)
1.3.6.1.4.1.2.267.10.4.4  (kyber-1024-90s-r3)

**Comment by baentsch** [comment]
Did you Check out #43 and #44? Is anyone looking AT them? @ounsworth you asked for them: Are they (or just one of them) OK for you?
